property_table.py

```diff
@@ -354,44 +354,45 @@ class PropertyTable:
             False,
             ["PartType5/SubgridMasses", "PartType5/Velocities"],
         ),
-        "BaryonInertiaTensor": (
-            "BaryonInertiaTensor",
+        "DarkMatterInertiaTensor": (
+            "DarkMatterInertiaTensor",
             6,
             np.float32,
             "kpc**2",
-            "3D baryon inertia tensor computed from the baryon (gas and stars) mass distribution, relative to the centre of potential. Stores diagonal components and one off-diagonal triangle as (1,1), (2,2), (3,3), (1,2), (1,3), (2,3).",
-            "baryon",
+            "3D inertia tensor computed iteratively from the DM mass distribution, relative to the halo centre. Diagonal components and one off-diagonal triangle as (1,1), (2,2), (3,3), (1,2), (1,3), (2,3).",
```

Comment:
Would it be useful to indicate somewhere how to recover the inertia tensor from this subset of the flattened representation?